tethys_services/backends/hydroshare_beta.py

```diff
@@ -0,0 +1,86 @@
+"""
+********************************************************************************
+* Name: hydroshare.py
+* Author: Nathan Swain and Ezra Rice
+* Created On: July 31, 2015
+* Copyright: (c) Brigham Young University 2015
+* License: BSD 2-Clause
+********************************************************************************
+"""
+from social.backends.oauth import BaseOAuth2
+
+class HydroShareOAuth2(BaseOAuth2):
```

Comment:
I think this class ought to be renamed HydroShareBetaOAuth2 and inherit from the HydroShareOAuth2 class, only overriding the necessary parameters. This would be a more DRY approach.